mod/wikirate/set/type/wikirate_company.rb

```diff
@@ -22,11 +22,10 @@ def view_caching?
   view :contribution_link do |args|
     no_of_related_metric = Card.search :type_id=>MetricID, :left=>card.name, :return=>"count"
     if no_of_related_metric > 0
-      card_link card.name+"+contribution",{:text=>"Contributions",:class=>"btn btn-default company-contribution-link"}
+      card_link card.name+"+contribution",{:text=>"View Contributions",:class=>"btn btn-primary company-contribution-link"}
```

Comment:
Line is too long. [123/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `+`.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.